File: app/src/main/java/com/example/unscramble/data/WordsDao.java
package com.example.unscramble.data;


import androidx.room.Dao;
import androidx.room.Insert;
import androidx.room.Query;

@Dao
public interface WordsDao {
    @Query("SELECT * FROM ITEMS")
    suspend fun getAllWords() : List<Words>

    @Insert
    suspend fun insert(vararg wordsModel: Words)
}

File: app/src/main/java/com/example/unscramble/data/WordsDao.kt
package com.example.unscramble.data

import androidx.room.Dao
import androidx.room.Insert
import androidx.room.Query

@Dao
interface WordsDao {

    @Query("SELECT * FROM items")
    suspend fun getAllWords() : List<WordsModel>

    @Insert
    suspend fun insert(vararg wordsModel: WordsModel)
}